src/invwarpedview.jl

```diff
@@ -0,0 +1,118 @@
+"""
+    InvWarpedView(img, tinv, [indices]) -> wv
+
+Create a view of `img` that lazily transforms any given index `I`
+passed to `wv[I]` to correspond to `img[inv(tinv)(I)]`. While
+technically this approach is known as backward mode warping,
+`InvWarpedView` is created using the forward transformation.
```

Comment:
Lets keep it here then. I like this additional layer of quality control. I also suspect that between `InvWarpedView` and `WarpedView`, the former is more convenient for a user to work with